### DISCUSSION TITLE
FSNotes 6.0.0

### DISCUSSION THREAD

**Discussion by glushchenko** [glushchenko/fsnotes#1411] [Announcements]
- Adds deep git integration (pull/push/commit message)
- Adds separate git repositories
- Adds folders encryption
- Adds folders reordering in sidebar
- Adds notes web sharing
- Adds TOC support
- Adds multiple windows support
- Adds quick notes
- Adds YAML heeaders parsing
- Adds clickable links option
- Adds windows frame and state saving
- Adds updated mermaid.js
- Adds new NSTextView icon
- Adds preview state saving for all notes
- Removes RTF support (you can still read old files)

And many more!

<hr /><em>This discussion was created from the release <a href='https://github.com/glushchenko/fsnotes/releases/tag/6.0.0'>FSNotes 6.0.0</a>.</em>

**Comment by iandol** [comment]
Wow, lots of new features, will take a while to digest. Thank you, and best wishes for you and your family in these difficult times!!!

**Comment by postprivee** [comment]
ios update in review?

**Reply by glushchenko** [reply]
> ios update in review?

No. Development not started yet.

**Comment by netluxe** [comment]
Per-folder encryption? Nice.

Just wanted to give my thanks for your continued hard work, developing/supporting this fantastic app.  Buying the upgraded iOS version was one of the best purchases I've ever made.

**Comment by tadamcz** [comment]
I received v6 on iOS today. Fantastic update, thanks @glushchenko and all contributors!
